docs/swagger.yaml

```
@@ -773,4 +782,25 @@
       summary: Gets a vehicle with uuid
       tags:
       - vehicle
+  /vehicle/change-owner:
+    put:
+      parameters:
+      - description: Dto for changing ownership
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)